doc/metadataGuide/guide.md

```diff
@@ -145,15 +144,15 @@ If blank nodes are used, then the the result should look similar to the followin
 ```
 Widoco will recognize the following properties when describing agents (persons or organizations):
 
-| Metadata category | Ontology annotation property |  Accepted property value | 
-| -- | -- | -- | 
-| Affiliation | [schema:affiliation](https://schema.org/affiliation), [org:memberOf](http://www.w3.org/ns/org#memberOf) | [Text](#text) or [Organization](#organization) or [BNode](#bnode)| [ontology](#onto), [config](#config)
-| Family Name | [schema:familyName](https://schema.org/familyName), [vcard:family-name](http://www.w3.org/2006/vcard/ns#family-name), [foaf:family_name](http://xmlns.com/foaf/0.1/family_name) | [Text](#text) 
-| Full name | [rdfs:label](http://www.w3.org/2000/01/rdf-schema#label), [schema:name](https://schema.org/name), [vcard:fn](http://www.w3.org/2006/vcard/ns#fn), [foaf:name](http://xmlns.com/foaf/0.1/name) | [Text](#text)
-| Given name | [schema:givenName](https://schema.org/givenName), [vcard:given-name](http://www.w3.org/2006/vcard/ns#given-name), [foaf:givenname](http://xmlns.com/foaf/0.1/givenname) | [Text](#text) 
-| Email | [schema:email](https://schema.org/email), [vcard:hasEmail](http://www.w3.org/2006/vcard/ns#hasEmail), [foaf:mbox](http://xmlns.com/foaf/0.1/mbox) | [Text](#text)  
-| URL | [schema:url](https://schema.org/url), [vcard:hasURL](http://www.w3.org/2006/vcard/ns#hasURL), [foaf:homepage](http://xmlns.com/foaf/0.1/homepage) | [URL](#url) 
- 
+|Metadata category|Ontology annotation property                                |Accepted property value            |
+|-----------------|------------------------------------------------------------|-----------------------------------|
+|Affiliation      |[schema:affiliation], [org:memberOf]                        |[Text] or [Organization] or [BNode]|
+|Family Name      |[schema:familyName], [vcard:family-name], [foaf:family_name]|[Text]                             |
+|Full name        |[rdfs:label], [schema:name], [vcard:fn], [foaf:name]        |[Text]                             |
+|Given name       |[schema:givenName], [vcard:given-name], [foaf:givenname]    |[Text]                             |
+|Email            |[schema:email], [vcard:hasEmail], [foaf:mbox]               |[Text]                             |
+|URL              |[schema:url], [vcard:hasURL], [foaf:homepage]               |[URI]                              |
+
```

Comment:
Sorry, why are you removing all the links here? I added them in the first place because having them at the end of the document is a pain to go back and forth. Plus, that way you can copy the metadata property directly from the link instead of having to go to the back to the document all the time. I am fine if you also want to have them at the end of the doc, but I am against removing them here

Reply:
They are separated just in the source code, to improve legibility of the table. But the [preview/rendering of the .md file](https://github.com/dgarijo/Widoco/blob/ae28862d06600d99dce7a54bcc1588fa4ba3fc93/doc/metadataGuide/guide.md) shows the links correctly at the place they're supposed to be.

Reply:
Oh, I was not aware of this way of setting up links. Will merge then. Thanks!